src/Symfony/Bundle/FrameworkBundle/Command/AboutCommand.php

```diff
@@ -28,6 +29,18 @@ class AboutCommand extends ContainerAwareCommand
 {
     /**
      * {@inheritdoc}
+     *
+     * @deprecated since version 3.4, to be removed in 4.0
+     */
+    protected function getContainer()
```

Comment:
the runtime deprecation is nice to have? it's here now :)

+1 for final, simply adding `@final` is sufficient? and make them realy final in 4.0.. right?